gen/KubernetesWatchGenerator/Program.cs

```diff
@@ -0,0 +1,250 @@
+using Newtonsoft.Json;
+using NJsonSchema;
+using NSwag;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KubernetesWatchGenerator
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            // Download the latest spec
+            const string kubernetesBranch = "v1.10.0";
```

Comment:
This is currently controlled by the `settings` file in the repo here:

https://github.com/kubernetes-client/csharp/blob/master/csharp.settings

We should read from that environment variable (or from that file) for consistency.